Dockerfile

```diff
@@ -1,35 +1,38 @@
+ARG NODE_VERSION=16.15.1-bullseye-slim
+
 # Use separate builder to retrieve & build node modules
-FROM node:16.15.1-slim AS builder
+FROM node:${NODE_VERSION} AS builder
 
 WORKDIR /usr/src/app
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         build-essential \
+        ca-certificates \
         git \
         jq \
         python3 \
-        ssh
+        ssh \
+        wget \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
We can `apt clean` instead - it's possible to to a dry run of that command to check if we're cleaning enough.

Reply:
according to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/, `apt clean` is supposed to be automatically called... but the exemple does the `rm -rf /var/lib/apt/lists/*`

so i choose to stick to the old & proven approach :)